frontend_tests/node_tests/dropdown_list_widget.js

```diff
@@ -38,7 +38,7 @@ run_test("basic_functions", () => {
 
     const {reset_button, $widget} = setup_zjquery_data(opts.widget_name);
 
-    const widget = dropdown_list_widget(opts);
+    const widget = new DropdownListWidget(opts);
```

Comment:
Ideally, I think you should squash this commit with the earlier one since that commit wouldn't pass CI alone.

Reply:
Oh Yeah! Silly me!